class-components/src/app/messages/en.json

```
@@ -0,0 +1,43 @@
+{
+  "utils": {
+    "Something went wrong, please try again later": "Something went wrong, please try again later",
+    "Retry": "Retry",
+    "Refresh": "Refresh",
+    "Rick and Morty": "Rick and Morty",
+    "No characters found": "No characters found"
+  },
+  "Nav": {
+    "Home": "Home",
+    "About app": "About app"
+  },
+  "NotFound": {
+    "Oops, there is nothing, probably Rick deleted this page": "Oops, there is nothing, probably Rick deleted this page",
+    "Go home": "Go home"
+  },
+  "About": {
+    "The application was made by": "The application was made by",
+    "as part of the": "as part of the",
+    "course": "course"
+  },
```

это просто ключи и значения

```suggestion
  "not-found": {
    "message": "Oops, there is nothing, probably Rick deleted this page",
    "go-home": "Go home"
  },
  "about": {
    "made-by": "The application was made by",
    "as-part": "as part of the",
    "course": "course"
  },
```

ну и структуру можно задавать различную типа 
/ pages
/ - / not-found
/ - / - / title
/ - / - / message
/ - / about
/ - / - / title
/ - / - / message
/ shared
/ - / go-home
/ - / confirm
/ - / close

и так далее